### DISCUSSION TITLE
Updating linux packages

### DISCUSSION THREAD

**Discussion by theRookieCoder** [gorilla-devs/ferium#211] [Announcements]
I've made some changes to the linux builds. Firstly, all the builds now use `musl` so `gcc-libs` is no longer required as a dependency. I've also moved away from GTK, only XDG is used for the GUI backend (but it does still work on GNOME) which works on most DEs afaik. Also there are now arm builds for linux. I've also changed the names of the zip files in the releases. All maintainers of ferium's linux packages (@KyleUltimate @Henryws @leo60228 @imsofi), please update your packages. As far as I know only bzip2 is a required runtime dependency for ferium now.

**Comment by soupglasses** [comment]
Already working under nixpkgs, NixOS/nixpkgs#194185

It seems really fast compared to last i used it. Does this new update come with speed improvements due to the musl change?

**Reply by theRookieCoder** [reply]
Interesting, I'm on a slow cellular connection at the moment so I wasn't sure. I've actually heard online that musl has less optimisations so it's much slower in benchmarks compared to GNU. It might also be caused by the Rust version.

**Reply by soupglasses** [reply]
Checking against `4.1.10`, it seems like i simply didn't remember just how fast ferium is. :sweat_smile:

**Reply by theRookieCoder** [reply]
Then it could also be your internet connection!

**Comment by theRookieCoder** [comment]
@KyleUltimate https://github.com/gorilla-devs/ferium/actions/runs/3173562611/jobs/5169317785
The AUR package completely broke and doesn't update anymore presumably because of the filename changes.

**Reply by JustSimplyKyle** [reply]
Ok, I will fix it tonight

**Reply by JustSimplyKyle** [reply]
Small question, does the xdg-desktop-portal and bzip2 are still required?

**Reply by theRookieCoder** [reply]
I think bzip2 would still be required, and yes XDG portal is now being used for GUI releases so it is required.

**Reply by JustSimplyKyle** [reply]
Hmm, since now you use musl, does `musl` the binary has to be installed?

**Reply by theRookieCoder** [reply]
Musl is an alternative to glibc and it's statically linked so that's not be necessary

**Reply by JustSimplyKyle** [reply]
Ok, now all binary sources has been updated!

**Reply by theRookieCoder** [reply]
Alright thank you!

**Reply by JustSimplyKyle** [reply]
Bonus, it now support aarch64!(only for binary packages)

**Comment by Elsie19** [comment]
I will update the pacstall package in a couple hours.

**Reply by Elsie19** [reply]
https://github.com/pacstall/pacstall-programs/pull/1607
